r2/r2/public/static/main.css

```diff
@@ -847,7 +847,7 @@ div.tools a.license:hover {
 
 a.read_more_link {
   color: rgb(50, 100, 146);
-  cursor: auto;
+  cursor: pointer;
```

Comment:
It's not clear to me why this was set to `auto`, since it's a link. I should probably delete this line instead of changing the value of the property. I'll make that change.